### DISCUSSION TITLE
Independant SQL Editor windows/results grid

### DISCUSSION THREAD

**Discussion by ilcaa72** [dbeaver/dbeaver#16773] [DBeaver Development]
i see you can have 2 SQL editor windows with results by selecting Window--> Editor-->toggle Split Editor vertically    BUT they are connected (at least mine is) meaning what you do to 1 sql editor you see in the other, its a mirror of each other....

Is there a way to disconnect them so they work independant?  Id like 2 SQL editor with their own results grid that work independant of each other...

please advise. thanks

**Comment by LonwoLonwo** [comment]
Hello @ilcaa72 

You can open two separate SQL Editors for your connection and then pull one of them to divide SQL Editor area. In this case two editors will be independent.

If this setting is enabled:

![2022-06-15 10_41_58-DBeaver Ultimate 22 1 0](https://user-images.githubusercontent.com/45152336/173771201-8a3a742f-11f4-4606-b9ed-3d4df842273d.png)

![oSv4Trc1OZ](https://user-images.githubusercontent.com/45152336/173771233-33c068bc-f4ef-47c0-a6af-a8bd9712e38a.gif)
